src/pages/EnvironmentsPage/environmentsPermissions.ts

```diff
@@ -8,19 +8,19 @@ import { getTerraUser } from 'src/libs/state';
 
 const makePermissionsProvider = (userEmailGetter: () => string): LeoResourcePermissionsProvider => {
   return <LeoResourcePermissionsProvider>{
-    canDeleteDisk: (disk: PersistentDisk) => {
+    hasDeletePermission: (disk: PersistentDisk) => {
```

Comment:
since this is specific to disks, this could maybe be `hasDeleteDiskPermission` or altered to take more types